grants_tagger/preprocess.py

```diff
@@ -34,14 +34,24 @@ def yield_preprocess_data(
     # cols = ['Grant Team', 'ERG', 'Lead Applicant', 'Organisation', 'Scheme', 'Title', 'Synopsis', 'Lay Summary', 'Qu.']
     processed_data = data.drop_duplicates(subset=["Grant_ID", "Sciencetags"])
     processed_data = processed_data.dropna(subset=["Synopsis"])
+    processed_data["intersection"] = processed_data["intersection"].str.replace("19: Maternal, Child", "19: Maternal Child")
+    processed_data["intersection"] = processed_data["intersection"].str.replace("24: Data Science, Computational", "24: Data Science Computational")
```

Comment:
when i do `make test` the `test_preprocess.py` tests fail. This seems to be because the 'intersection' key isnt in the data for the tests? i dont think `tagger 1 only` is in them either. I wonder whether it's worth doing a processed_data.get('intersection') or should intersection just always be in the data being processed?

Reply:
should it always be? that's a good question, probably not. let me think about it.